persistence/sql/migrations/sql/20230104193739000000_courier_list_index.up.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX courier_messages_nid_created_at_id_idx ON courier_messages (nid, created_at, id);
+
+CREATE INDEX courier_messages_recipient_at_id_idx ON courier_messages (recipient);
```

Comment:
The `status` variant was already added earlier in `persistence/sql/migrations/sql/20210307130558000000_courier_status_index.cockroach.up.sql`